src/components/checkbox/checkbox.js

```diff
@@ -1,29 +1,144 @@
-function listener(event) {
-  const element = event.target;
-  if (element.tagName === 'INPUT' && element.type === 'checkbox' && element.hasAttribute('checked') !== element.checked) {
-    if (element.checked) {
-      element.setAttribute('checked', '');
+import mixin from '../../globals/js/misc/mixin';
+import createComponent from '../../globals/js/mixins/create-component';
+import InitComponentBySearch from '../../globals/js/mixins/init-component-by-search';
+// import default from '../lightbox/lightbox';
+
+const stateChangeTypes = {
+  checked: 'true',
```

Comment:
From what I read https://github.com/alisonjoseph/carbon-components/blob/3b8a6240df68dca2911abd27b18be93994408a58/src/components/checkbox/checkbox.js#L72, did you mean `checked`/`unchecked` for the property values here, too?

Reply:
We have this line here `this.element.setAttribute('aria-checked', state);` on line 76 and the correct values are true, false or mixed. https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/checkbox/checkbox-1.html

Reply:
Thank you for clarifying that those values are from `aria-checked` attribute - Given that, let’s make the error message [there](https://github.com/alisonjoseph/carbon-components/blob/3b8a6240df68dca2911abd27b18be93994408a58/src/components/checkbox/checkbox.js#L72) in sync (in addition to `console.log()` to `throw new Error()` change). Thanks! 😊